### ISSUE TITLE
Mall till ER modellering

### ISSUE THREAD

**Issue by mosbth** [dbwebb-se/databas#44]
Jag hittade en gammal mall till ER-modelleringsrapporten i kmom03. Men så blev jag osäker på om den verkligen används...?

Här länken till mallen iallafall.
https://docs.google.com/document/d/1Fr8YQrloF0lyXHszwPA06qhMreSZom5zyemBfszXOR0/edit?usp=sharing

Låt mig veta om den hjälper eller om det bara kändes onödigt.

**Comment by hiwaf** [comment]
Får vi använda din text i "Beskriv databasen i ett textstycke" som om det vore text från en kund och utgå från den om man kör BuckStar online shopping?

**Comment by mosbth** [comment]
Ja, det går bra. Det är ju samma sak som att kopiera texten från uppgiften och lyfta in den i dokumentet. Så är tanken.

Det är också fritt att bearbeta texten om man vill justera lite eller lägga till saker för att förtydliga.
